core/src/main/java/org/apache/calcite/rex/RexLiteral.java

```diff
@@ -785,6 +785,31 @@ public Comparable getValue4() {
     }
   }
 
+  /**
+   * Returns the value of this literal, in the form of GeodeFilter
+   * translator wants it.
+   */
+  public Object getValue5() {
```

Comment:
That's interesting.
@asereda-gs , should `Geode`-specific bits be added to `RexLiteral`?
It looks weird to me.

Reply:
Deleted the RexLiteral.java file change

Reply:
@vlsi I think it was an oversight. This PR shouldn't change anything in `core` module (only `geode`).